signer/signer.go

```diff
@@ -122,6 +122,8 @@ func (s *GenericSigner) Sign(ctx context.Context, desc ocispec.Descriptor, opts
 		SigningTime:   time.Now(),
 		SigningScheme: signature.SigningSchemeX509,
 		SigningAgent:  signingAgentId,
+		Timestamper:   opts.Timestamper,
+		TSARootCAs:    opts.TSARootCAs,
```

Comment:
Is `TSARootCAs` optional or mandatory parameter when Timestamper is specified?

Reply:
This is a good catch. It's `mandatory`. Adding a validation to the `signRequest` to enforce this.

Reply:
I am still debating about making it mandatory but to start with lets make it mandatory, we can always relax this requirement later.